src/chat/_chatwidget.py

```diff
@@ -32,6 +32,8 @@ class ChatWidget(FormClass, BaseClass, SimpleIRCClient):
     irc_host = Settings.persisted_property('chat/host', type=str, default_value='irc.' + defaults['host'])
     irc_tls = Settings.persisted_property('chat/tls', type=bool, default_value=False)
 
+    autoJoinChannels = Settings.persisted_property('chat/autoJoinChannels', type=list, default_value=[])
```

Comment:
I'd prefer we used under_scores rather than camelCase.

Reply:
Ok